resume-project/src/components/ContactItem.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const ContactItem = (props) => (
```

Comment:
you can do `const ContactItem = ({iconName, linkAction, linkTarget}) => ` to avoid `props.linkTarget` all the time